src/xpanse-api/generated/types.gen.ts

```diff
@@ -0,0 +1,5475 @@
+/*
```

Comment:
can you please check this file again? i see a big different file when i generate with `enum: typescript`